tests/e2e/configurer/chain/node.go

```diff
@@ -58,10 +58,12 @@ func (n *NodeConfig) Run() error {
 		return err
 	}
 
+	n.rpcClient = rpcClient
+
 	require.Eventually(
 		n.t,
 		func() bool {
-			if _, err := rpcClient.Health(context.Background()); err != nil {
+			if _, err := n.QueryCurrentHeight(); err != nil {
```

Comment:
On another look, what difference does it make?

`QueryCurrentHeight` makes an RPC call under the hood

cc: @czarcas7ic

Reply:
Do you know why we were defining n.rpcClient after the Eventually check?

To answer your question, I have had issues with the health endpoint in the past, whereas the current height check has never failed me. This change was less based in science and more based on gut feeling...that being said I haven't seen a false fail yes 😅 *vigorously knocks on wood*

Reply:
> Do you know why we were defining n.rpcClient after the Eventually check?

I don't think it matters where we make an assignment in the original version. In the new one, we do have to do it before the require because that's what `n.QueryCurrentHeight` uses